org_fedora_oscap/rule_handling.py

```diff
@@ -170,10 +184,8 @@ def new_rule(self, rule):
         first_word = rule.split(None, 1)[0]
         try:
             actions[first_word](rule)
-        except KeyError:
```

Comment:
We still need to catch `KeyError`. This catch is about not finding the rule in `actions` dictionary.

Reply:
@jan-cerny per @yuumasato. Let's make sure to check for `KeyError`.

Reply:
thank you

Reply:
It will be here: https://github.com/OpenSCAP/oscap-anaconda-addon/pull/51